js/interface.js

```diff
@@ -1365,6 +1382,17 @@ $('#show-access-rules').click(function () {
   $('.empty-data-source-rules').toggleClass('hidden', currentDataSourceRules.length > 0);
   $('#access-rules-list table').toggleClass('hidden', !currentDataSourceRules.length);
 
+  function operatorDescription(operation) {
+    switch(operation) {
+      case 'equals':
+        return 'equals to';
+      case 'notequals':
+        return 'does not equals to';
+      case 'contains':
+        return 'contains';
+    }
```

Comment:
Let's add a `default` just in case, which should throw an error (because backend won't be able to process it).

Reply:
@squallstar I thought about simply returning the operation as its description too. But if we hadn't defined a description, then there's a chance that the operation name isn't supported by the backend. Shouldn't it throw an error instead of letting users think it'll be OK?

Reply:
I don't think we should throw an error. If we ever have other places that can save new rules in the future than the UI should basically show a "non localised label" rather than failing.

Reply:
I'm OK with this. FYI if something was suggested, it'll be helpful to provide a reason if the suggestion isn't accepted.